syntax = "proto2";

package fyBatteryGroupTemperatureInfo; //电池组温度信息

message BatterySingleTemperature //温度探头信息
{
        required uint32 addr_number = 1;       //温度探头个数
        repeated int32 temperature = 2;        //所有探头的温度值
}

